data/agent.conf

```diff
@@ -0,0 +1,5 @@
+[fwupdagent]
+
+# The server hostname, port and path, e.g.
+# https://localhost:8081/fwdash
```

Comment:
I think you should add some comments into this file to indicate that this is intended for Enterprise use and most people shouldn't touch it.  Since this is going to be installed on all systems with fwupd most people who see it will have no idea what it is.

Reply:
Added, thanks.